### PR TITLE
Analysis for EAP7-1021, CLI key navigation

### PR DESCRIPTION
**Pull request by jfdenise**

### CODE REVIEW

File: cli/EAP7-1021_Command_Navigation.adoc [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+* Ctrl+A or HOME: Linux, Windows, Solaris, Mac OSX
+
+=== Go to the end of the line
+
+* Ctrl+E or END: Linux, Windows, Solaris, Mac OSX
+
+=== Go left (back) one character
+
+* Ctrl+B: Linux, Windows, Solaris, Mac OSX
```
Comment (dudaerich) [inline cli/EAP7-1021_Command_Navigation.adoc]: I know that this is obvious but to be fully complete there should be "or Arrow left". The same applies for Go right (forward) one character.

Comment (jfdenise) [reply cli/EAP7-1021_Command_Navigation.adoc]: Good catch, I updated for both left/right arrow.

**Comment by jfdenise** [comment]
@rsvoboda , tested on HP-UX and updated analysis.

**Comment by jfdenise** [comment]
@dudaerich , I updated the analysis to state that HOME/END key are not supported for Mac OSX, these keys go to beginning/end of document (terminal in our case).

**Comment by dudaerich** [comment]
@jfdenise , did you push the changes? I don't see any mention that HOME/END key are not supported for Mac OSX.

**Comment by jfdenise** [comment]
@dudaerich , pushed.

**Comment by dudaerich** [comment]
@jfdenise I've reviewed the change and it's ok for me. Thanks.

**Comment by kabir** [comment]
@jfdenise Can you please add in the new Community Documentation section I just mentioned on wildfly-dev?

**Comment by jfdenise** [comment]
@kabir , done. Sorry for that.

**Pull request merged**